servers/rendering/renderer_rd/environment/sky.cpp

```diff
@@ -1286,6 +1280,16 @@ void SkyRD::update_radiance_buffers(Ref<RenderSceneBuffersRD> p_render_buffers,
 		sky_mode = RS::SKY_MODE_QUALITY;
 	}
 
+#if defined(MACOS_ENABLED) && defined(__x86_64__)
```

Comment:
Is this check needed every frame for every sky? It should just be done once right?

Reply:
Moved it to a separate function, now it should be called only one.